src/app/_lib/posthog.ts

```
@@ -14,6 +14,14 @@ export type FlagsMatcher = Record<string, undefined | {
 
 export const FeatureFlags = {
     IS_APP_DISABLED: 'is_app_disabled',
+    INVITE_ONLY: 'invite-only',
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'INVITE_ONLY' should be before 'IS_APP_DISABLED'.

(sort-keys-fix/sort-keys-fix)